File: Module 3/3A Javascript/JavaScript Coding Challenge/Task10.js
// Create a function that filters out negative numbers

function filterNegativeNumbers(arr) {
    return arr.filter(num => num >= 0);
}

// Sample usage
let numberArray = [3, -1, 4, -1, 5, -9, 2, -6, 5, 3, 5];
let filteredArray = filterNegativeNumbers(numberArray);
console.log('Original Array:', numberArray);
console.log('Filtered Array:', filteredArray);

File: Module 4/4C_Everyday_Go_To_Python/Python_Coding_Challenge.py/Task10.py
"""
Create a function that filters out negative numbers.
"""
def filter_negative_numbers(numbers):
    return [num for num in numbers if num >= 0]

# Example usage
numbers_list = [-10, 15, -3, 20, 0, -7, 8]
filtered_list = filter_negative_numbers(numbers_list)

print("Original List:", numbers_list)
print("Filtered List (No Negatives):", filtered_list)
